lib/edit-site-page.php

```diff
@@ -208,6 +208,28 @@ function gutenberg_edit_site_init( $hook ) {
 	global $post;
 	$settings = apply_filters( 'block_editor_settings', $settings, $post );
 
+	// Preload block editor paths.
+	// most of these are copied from edit-forms-blocks.php.
+	$preload_paths = array(
+		'/',
+		'/wp/v2/types?context=edit',
+		'/wp/v2/taxonomies?per_page=-1&context=edit',
+		'/wp/v2/themes?status=active',
+		sprintf( '/wp/v2/templates/%s?context=edit', $_wp_current_template_id ),
+		'/wp/v2/users/me?post_type=templates&context=edit',
```

Comment:
does having `post_type=templates` make sense here?  For the site editor isn't our post type `undefined` ?

Reply:
I think our post type is wp_template actually, so yeah that might be the only unused entry here, I'll test it.